src/rabbit_amqqueue.erl

```diff
@@ -609,6 +622,21 @@ consumers_all(VHostPath) ->
                   {ChPid, CTag, AckRequired, Prefetch, Args} <- consumers(Q)]
           end)).
 
+consumers_all(VHostPath, Ref, AggregatorPid) ->
+    ConsumerInfoKeys=consumer_info_keys(),
+    map(list(VHostPath),
+        fun (Q) ->
+                AggregatorPid !
+                    {Ref, [lists:zip(
```

Comment:
can this be extracted into a function to clarify a bit code intention. After looking at it I do understand that you are matching ConsumerInfoKeys with their values, but still, I can't get that info at first sight without stoping at inspecting the code.